wp-content/themes/csisnuclearnetwork/assets/_scss/pages/single.scss

```diff
@@ -227,14 +227,11 @@
   }
 
   &__footer {
+    max-width: $size__content-max-width;
```

Comment:
This should actually be `900px` [wide](https://www.figma.com/file/cAg0A1ww0s2KbJdIbUvl2j/PONI-Redesign-Handoff?node-id=131%3A43178).  You can make a new width variable for the footer in `_variables.scss` for this section.